parser/declaration.go

```diff
@@ -645,13 +645,14 @@ func parsePragmaDeclaration(p *parser) (*ast.PragmaDeclaration, error) {
 //
 //	importDeclaration :
 //	    'import'
-//	    ( identifier (',' identifier)* 'from' )?
+//	    ( identifier (as identifier)? (',' identifier (as identifier)?)* 'from' )?
```

Comment:
The `as` keyword is a token, so it needs to be quoted:

```suggestion
//	    ( identifier ('as' identifier)? (',' identifier ('as' identifier)?)* 'from' )?
```